<?xml version="1.0" ?>
<launch>

  <arg name="config"		default="aist"/>
  <arg name="robot_name"	default="b_bot"/>
  <arg name="speed"		default="0.1"/>
  <arg name="ns"		default="/"/>

  <node ns="$(arg ns)" name="tool_calibration"
	pkg="aist_handeye_calibration" type="tool_calibration.py"
	output="screen">
    <rosparam file="$(find aist_routines)/config/$(arg config)_config.yaml"/>
    <param name="robot_name"	value="$(arg robot_name)"/>
    <param name="speed"		value="$(arg speed)"/>
  </node>

</launch>
